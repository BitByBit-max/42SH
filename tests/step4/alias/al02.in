alias caca='echo caca' coco='echo coco'

alias ahoj='echo ahoj'

alias
