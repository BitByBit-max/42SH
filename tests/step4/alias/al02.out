ahoj='echo ahoj'
caca='echo caca'
coco='echo coco'
